js/dom.js

```diff
@@ -0,0 +1,65 @@
+const container = document.querySelector('#books-list');
+
+function removeBook(books, index) {
+  const art = document.querySelector(`#article-${index}`);
+  art.remove();
+  books.splice(index, 1);
+  localStorage.setItem('books', JSON.stringify(books));
+}
+
+function createArticle(books, index) {
+  const article = document.createElement('article');
+  article.setAttribute('id', `article-${index}`);
+  const title = document.createElement('p');
+  title.textContent = books[index].title;
+  article.appendChild(title);
+  const author = document.createElement('p');
+  author.textContent = books[index].author;
+  article.appendChild(author);
+  const button = document.createElement('button');
+  button.classList.add('remove-btn');
+  button.textContent = 'Remove';
+  button.addEventListener('click', () => { removeBook(books, index); });
+  article.appendChild(button);
```

Comment:
- [ ] Please use the best practice rules in ES6, you should use **arrow function** please.